thorlcr/activities/join/thjoinslave.cpp

```diff
@@ -458,7 +457,10 @@ class JoinSlaveActivity : public CSlaveActivity, implements ILookAheadStopNotify
             leftStream.setown(iLoaderL->load(leftInputStream, abortSoon));
             isemptylhs = 0 == iLoaderL->numRows();
             stopLeftInput();
+
+            CRuntimeStatisticCollection spillStats(spillStatistics);
```

Comment:
We should introduce
mergeStats(spillStats, iLoadedL, spillStatistics) to avoid the temporary StatsCollection.
Created HPCC-24004 to implement once this is merged